Travel Scheduler/Test/APITesting.m

```diff
@@ -8,6 +8,8 @@
 
```

Comment:
In general, for this file, if this is a unit testing initiative, I absolutely LOVE it.
Possibly modifying it such that it has asserts would be a great move. So if you have something like:

`if (x) {`
`    NSLog(@"Yes");`
`} else {`
`    NSLog(@"No");`
`}`

Replace with:

`<assert_method>(x)`